src/inc/sift-events/sift-events.php

```diff
@@ -626,6 +626,7 @@ public static function update_or_create_order( string $order_id, \WC_Order $orde
 			'$site_country'    => wc_get_base_location()['country'],
 			'$ip'              => self::get_client_ip(),
 			'$time'            => intval( 1000 * microtime( true ) ),
+			'product_channel'  => 'woocommerce-com',
```

Comment:
Please keep in mind SfW is meant to be platform agnostic. So hardcoding this value is not the way to go.
Also, this is a custom event, and should be treated as well.

We already [have a filter ](https://github.com/woocommerce/sift-for-woocommerce/blob/trunk/src/inc/sift-events/sift-events.php#L875C3-L877C1) for this specific purpose.

So the fix should be on WCCOM sidecar plugin and check for $create_order / $update_order and add the appropriated `product_channel` properties.